tests/Integration/Mail/SendingMailWithLocaleTest.php

```diff
@@ -77,7 +77,7 @@ public function test_mail_is_sent_with_locale_updated_listeners_called()
 
         Mail::to('test@mail.com')->locale('es')->send(new TimestampTestMail);
 
-        $this->assertContains('nombre dentro de 1 día',
+        $this->assertRegExp('/nombre (en|dentro de) (un|1) día/',
```

Comment:
Why have these two tests changed in this PR?

Reply:
Why not? Carbon 2 is a major release, it has major changes such as translations harmonization with momentjs. It comes with output changes.